flopy/mf6/mfpackage.py

```diff
@@ -2446,7 +2465,7 @@ def __setattr__(self, key, value):
             package = self._packages[0]
             setattr(package, key, value)
             return
-        super().__setattr__(key, value)
+        super(MFChildPackages, self).__setattr__(key, value)
```

Comment:
I see.  This must be why you added object.  But isn't there another way to call super in this case?

Reply:
This was changed by black during linting. Either method works fine.

Reply:
I can revert these four changes if you'd prefer.

Reply:
I guess it doesn't really matter to me.  But @mwtoews recently removed all occurrences of "object" in order to simplify things because they're not needed anymore.  In poking around the googles I didn't find anything to indicate why black was doing this and/or why it might be necessary in some cases.  If it's ready to merge now, i say go ahead, and then maybe @mwtoews can weigh in on the best way to handle this.

Reply:
@langevin-usgs This is ready to merge. I can spend a few minutes googling around to see why black might have done this.

Reply:
@jlarsen-usgs bizarre, what do you get with `black --version`? It might be an older behavior.

Reply:
@mwtoews I'm using black 20.8b1 right now. It looks like the most recent is 20.8b2

Reply:
CI is running 20.8b1 too, but leaves these unchanged. Probably some old code from 9c62b8bf0c789f65ea7ebaef84710b6fbcd695b8 (the first commit in this PR) was brought back in.

Reply:
@mwtoews Your explanation seems to make more sense and upon closer inspection, there were additional differences between this PR and develop in mfpackage.py that I did not make that show up in that commit. I just went back through mfobservation.py and mfpackage.py and changed those code blocks to match the develop branch.